pkg/controller/kfservice/kfservice_controller.go

```diff
@@ -124,7 +124,7 @@ func (r *ReconcileService) Reconcile(request reconcile.Request) (reconcile.Resul
 
 	desiredService, err := resources.CreateKnativeService(kfsvc)
 	if err != nil {
-		log.Error(err, "Failed to create desired ksvc", "name", desiredService.Name)
+		log.Error(err, "Failed to create desired Knative Serving Service", "name", kfsvc.Name)
```

Comment:
```suggestion
		log.Error(err, "Failed to create desired Knative Serving Service", "name", desiredService.Name)
```

Reply:
when `err != nil`,  maybe we have no idea if desiredService is nil or not (although current resources.CreateKnativeService logic returing desiredService is not nil, even err will never be nil), so I think use `kfsvc.Name` instead. Make sense?

Reply:
yes, you are right.